### DISCUSSION TITLE
[Feature request] Different opacities for panes and margins

### DISCUSSION THREAD

**Discussion by raj-magesh** [kovidgoyal/kitty#4250] [Ideas]
It would be nice to have separate opacity control for each pane (or even all panes), separate from the background opacity. Specifically, I'd like to control the opacity of text region + padding (inside the boundary) independent of the opacity of the margin (outside the boundary).

Is this possible using the existing features? If not, would you be interested in implementing it or consider adding it should someone send a PR?

**Comment by kovidgoyal** [comment]
On Wed, Nov 17, 2021 at 04:46:42AM -0800, Raj wrote:
> It would be nice to have separate opacity control for each pane (or even all panes), separate from the background opacity. Specifically, I'd like to control the opacity of text region + padding (inside the boundary) independent of the opacity of the margin (outside the boundary).

No.

> 
> Is this possible using the existing features? If not, would you be interested in implementing it or consider adding it should someone send a PR?

Would depend on the details of the PR. If it involves too much overhead
either performance or code complication wise, probably not.
